src/main/java/medicaldoctor/ParentInterface.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package medicaldoctor;
+
+/**
+ *
+ * @author Salmanhussain
```

Comment:
I have been removing author since, I suppose we should agree on if we want to include these or not.